app/main.py

```diff
@@ -1,3 +1,40 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        return Distance(self.km + other.km)
```

Comment:
`other` can be of the int type.
In that case,  `other` hasn't attribute `km`.
If you don't know how to solve this problem ask in `#py_chat`

Reply:
Check Please